packages/splits-sdk-react/src/index.ts

```diff
@@ -63,4 +63,12 @@ export {
   usePassThroughTokens,
   usePassThroughWalletPause,
   usePassThroughWalletExecCalls,
+  useCreateSplitV2,
+  useDistributeTokenV2,
+  useBatchWithdrawWarehouse,
+  useExecCalls,
+  useSetPause,
+  useTransferOwnership,
+  useUpdateSplitV2,
+  useWithdrawWarehouse,
```

Comment:
nit: wondering if we should start to export these by type or something.

so someone could do `import { useCreateSplitV2, useDistributeTokenV2 } from '@0xsplits/splits-sdk-react/splitsV2`

anyway...don't think you need to do that in this pr. i might make a ticket for that now though, i do think that feels like a nice change. feels like it's a bit unwieldy just exporting everything together.

Reply:
https://linear.app/splits/issue/PE-1950/react-sdk-export-by-type